.mergify.yml

```diff
@@ -0,0 +1,26 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "#approved-reviews-by>2"
+
+pull_request_rules:
+  - name: automerge to main with label automerge and branch protection passing
+    conditions:
+      - "#approved-reviews-by>2"
+      - base=main
+      - label=A:automerge
+    actions:
+      queue:
+        name: default
+        method: squash
+        commit_message_template: |
+          {{ title }} (#{{ number }})
+          {{ body }}
```

Comment:
do we need that `(#{{ number }})` there at all? According to [GitHub documentation for squash merges](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#merge-message-for-a-squash-merge), if you have more than one commit, the title of the PR is used along with the number by default:

> The pull request title, followed by the pull request number

the story is different if the PR is composed of just 1 commit, the title of the commit is used instead.